api/nodecontrol/lameduck-response.json

```diff
@@ -4,15 +4,15 @@
   "title": "LameduckResponse",
   "type": "object",
   "properties": {
-    "node_id": {
+    "id": {
       "type": "string",
-      "description": "The unique identifier of the node"
+      "description": "The unique identifier of the actor"
```

Comment:
the outside world should never know about the existence of actors. This ID probably doesn't need to be here at all, since the lame duck request would've been sent to a specific node.